src/test/scala/it/pagopa/interop/attributeregistryprocess/AttributeRegistryApiServiceSpec.scala

```diff
@@ -73,9 +72,9 @@ class AttributeRegistryApiServiceSpec
             id = UUID.randomUUID(),
             code = Some("YADA"),
             kind = PersistentAttributeDependency.Certified,
-            description = "YADA",
+            description = "test",
```

Comment:
I expected this to remain the same, right?

Reply:
You are not an human being.